src/Plugins/ComplexCore/test/ScalarSegmentFeaturesTest.cpp

```diff
@@ -0,0 +1,134 @@
+
+#include <catch2/catch.hpp>
+
+#include "complex/DataStructure/Geometry/TriangleGeom.hpp"
+#include "complex/Parameters/ArrayCreationParameter.hpp"
+#include "complex/Parameters/BoolParameter.hpp"
+#include "complex/Parameters/MultiArraySelectionParameter.hpp"
+#include "complex/UnitTest/UnitTestCommon.hpp"
+#include "complex/Utilities/DataArrayUtilities.hpp"
+#include "complex/Utilities/Parsing/HDF5/H5FileWriter.hpp"
+
+#include "ComplexCore/ComplexCore_test_dirs.hpp"
+#include "ComplexCore/Filters/ScalarSegmentFeatures.hpp"
+
+using namespace complex;
+using namespace complex::UnitTest;
+using namespace complex::Constants;
+namespace Reconstruction
+{
+std::shared_ptr<DataStructure> CreateEbsdTestDataStructure()
```

Comment:
```suggestion
DataStructure CreateEbsdTestDataStructure()
```
Doesn't need to be `std::shared_ptr`.